data/json/bionics.json

```diff
@@ -57,6 +72,10 @@
         "type": "bionic",
         "id": "bio_metabolics",
         "name": "Metabolic Interchange",
+        "occupied_bodyparts": [
+            [ "TORSO", 5 ],
+            [ "HEAD", 2 ]
```

Comment:
Why head?

Also, it has a much lower cost than other recharger bionics, despite being (arguably) the best one.